sklearn/calibration.py

```diff
@@ -63,8 +63,9 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin):
         - An iterable yielding train/test splits.
 
         For integer/None inputs, if ``y`` is binary or multiclass,
-        :class:`StratifiedKFold` used. If ``y`` is neither binary nor
-        multiclass, :class:`KFold` is used.
+        :class:`sklearn.model_selection.StratifiedKFold` is used. If ``y`` 
```

Comment:
:class:`StratifiedKFold`

should be enough. At least in cross_val_predict it's enough. I don't think we should expand
full import.